shell/platform/darwin/macos/framework/Source/FlutterDartProject.mm

```diff
@@ -81,6 +81,13 @@ - (NSString*)ICUDataPath {
   return path;
 }
 
+- (NSString*)applicationLibraryPath {
+  if (_dartBundle == nil) {
+    return nil;
+  }
+  return [_dartBundle executablePath];
+}
```

Comment:
Sending a message to nil always evaluates to nil, so this can just be:
```objc
- (NSString*)applicationLibraryPath {
   return [_dartBundle executablePath];
 }
```